tasks/compile-javascripts.mjs

```diff
@@ -30,6 +30,11 @@ export async function compileJavaScripts () {
   } catch (cause) {
     throw new PluginError('compile:js', cause)
   }
+
+  // Write VERSION.txt file
+  if (isDist) {
+    await writeFile(join(destination, 'VERSION.txt'), pkg.version + EOL)
+  }
```

Comment:
**issue** I'm not sure it should be the responsibility of the JavaScript compilation to write the 'VERSION.txt' (nor the CSS compilation, or the compilation itself). We had a neat separation of concerns by having it as a separate task and I'm don't think there's any harm in keeping it that way (especially as it's a tangent matter to compiling rollup).

**suggestion** The existing asset-version task would benefit from the new way to access the version package that you added though 🙌🏻

Reply:
Ah don't worry, I can put back some of those 400+ removed lines of code 😉

Reply:
Cheers 😄

Reply:
Moved it here:

* https://github.com/alphagov/govuk-frontend/pull/3114